gradle.properties

```diff
@@ -2,7 +2,7 @@ org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
 version=1.0.10-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha7-SNAPSHOT
+ballerinaLangVersion=2.0.0-alpha7-20210326-211300-6ccfb47f
```

Comment:
```suggestion
ballerinaLangVersion=2.0.0-alpha7-20210327-165900-8a5ba56f
```

Reply:
We can't change only the transaction's lang version. Need to update all the dependencies' lang version. Therefore, the build failed with NPE.